showcase/app/components/mock/components/form/super-select/selected-component-multiple.gts

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
[question] would a simple approach like this one https://github.com/hashicorp/design-system/blob/main/showcase/app/components/mock/app/main/generic-text-content.gts work in this case?